grpc-proto/src/main/java/io/stargate/grpc/Values.java

```diff
@@ -163,4 +165,143 @@ public static Value of(Map<Value, Value> fields) {
         .setCollection(Collection.newBuilder().addAllElements(values).build())
         .build();
   }
+
+  public static boolean bool(Value value) {
+    if (value.getInnerCase() != InnerCase.BOOLEAN) {
+      throw new IllegalArgumentException("Expected boolean value");
+    }
+    return value.getBoolean();
+  }
+
+  public static int int_(Value value) {
+    if (value.getInnerCase() != InnerCase.INT) {
+      throw new IllegalArgumentException("Expected int value");
+    }
+    int intValue = (int) value.getInt();
+    if (intValue != value.getInt()) {
+      throw new IllegalArgumentException(
+          String.format("Valid range for int is %d to %d", Integer.MIN_VALUE, Integer.MAX_VALUE));
+    }
+    return intValue;
+  }
+
+  public static long bigint(Value value) {
+    if (value.getInnerCase() != InnerCase.INT) {
+      throw new IllegalArgumentException("Expected int value");
+    }
+    return value.getInt();
+  }
+
+  public static short smallint(Value value) {
+    if (value.getInnerCase() != InnerCase.INT) {
+      throw new IllegalArgumentException("Expected int value");
+    }
+    short shortValue = (short) value.getInt();
+    if (shortValue != value.getInt()) {
+      throw new IllegalArgumentException(
+          String.format("Valid range for int is %d to %d", Short.MIN_VALUE, Short.MAX_VALUE));
+    }
+    return shortValue;
+  }
+
+  public static byte tinyint(Value value) {
+    if (value.getInnerCase() != InnerCase.INT) {
+      throw new IllegalArgumentException("Expected int value");
+    }
+    byte byteValue = (byte) value.getInt();
+    if (byteValue != value.getInt()) {
+      throw new IllegalArgumentException(
+          String.format("Valid range for int is %d to %d", Byte.MIN_VALUE, Byte.MAX_VALUE));
+    }
+    return byteValue;
+  }
+
+  public static float float_(Value value) {
+    if (value.getInnerCase() != InnerCase.FLOAT) {
+      throw new IllegalArgumentException("Expected float value");
+    }
+    return value.getFloat();
+  }
+
+  public static double double_(Value value) {
+    if (value.getInnerCase() != InnerCase.DOUBLE) {
+      throw new IllegalArgumentException("Expected double value");
+    }
+    return value.getDouble();
+  }
+
+  public static ByteBuffer byteBuffer(Value value) {
+    if (value.getInnerCase() != InnerCase.BYTES) {
+      throw new IllegalArgumentException("Expected bytes value");
+    }
+    ByteBuffer bytes = ByteBuffer.allocate(value.getBytes().size());
```

Comment:
Does `getBytes()` always return a new copy? If so, would be good to use temporary variable to hold on to that copy to avoid extra copy.
(if method returns raw value instance, does not matter)

Reply:
it doesn't return a copy.

Reply:
Ok then that's fine.